File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfTableStyle.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import java.util.Map;

import com.neome.api.meta.base.Types.MetaIdTableStyle;
import com.neome.api.meta.base.dto.StudioBase;
import com.neome.api.meta.base.dto.StudioDtoTableStyle;

@SuppressWarnings("unused")
public class StudioMapOfTableStyle extends StudioBase
{
  public MetaIdTableStyle[] keys;

  public Map<MetaIdTableStyle, StudioDtoTableStyle> map;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioMapOfTableStyle.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.MetaIdTableStyle
import com.neome.api.meta.base.dto.StudioBase
import com.neome.api.meta.base.dto.StudioDtoTableStyle

interface StudioMapOfTableStyle : StudioBase
{
  val keys: List<MetaIdTableStyle>
  val map: Map<MetaIdTableStyle, StudioDtoTableStyle>
}